webapp/src/client/app/modules/web-client/vnc/web-client-vnc.component.ts

```diff
@@ -118,6 +118,17 @@ export class WebClientVncComponent extends WebClientBaseComponent implements OnI
     },
   ];
 
+  sliders = [
+    {
+      label: 'Wheel Speed',
+      value: 1,
+      onChange: (value: number) => this.setWheelSpeedFactor(value),
+      min: 0.1,
+      max: 5,
```

Comment:
question/suggestion: do you think a max of `2` or `3` would be reasonable? 5 times faster that the default sounds a bit much, but I didn’t try on my side.

Reply:
Done